micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/components/okhttp/OkHttpConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package io.micrometer.spring.samples.components.okhttp;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.spring.autoconfigure.MetricsProperties;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+@Configuration
+public class OkHttpConfiguration {
+
+
+	private final MeterRegistry meterRegistry;
+	private final MetricsProperties metricsProperties;
+
+	public OkHttpConfiguration(MeterRegistry meterRegistry, MetricsProperties metricsProperties) {
```

Comment:
Since these are only used for the bean you can remove them from here, and just set them on the `Bean`'s method

Reply:
Fixed